_posts/3420-10-3-Marcus-Jones.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title:  "October 3th 3420 | Marcus Jones | USS Absurdly Dignified"
+---
+![Image of Captain Patterson and Captain Ray]
+(https://nms-seventh-fleet.github.io/images/image21.png)
+
+<p>Another surprise today—Captain Patterson has returned to the local cluster! We rendezvoused in the Makvabar system. He still hasn't found a cure for the alien mutagen that turned him into a horrible fish-man, but otherwise he seemed in good health.</p>
+
+<p>I quizzed Patterson on his knowledge of the ancient portal glyphs, hoping he might have picked up some tidbits on his travels. Unfortunately, he knew even less about them than I did.</p>
+
+<p>Disappointed, I joined Patterson in surveying several new planets, including Eithon, a garden world in the Vodanda-Ilib XII system. I am astonished by how many of these planets we've overlooked in our own backyard. The local economy is only so-so, but the weather is fine and the wildlife is particularly striking.</p>
+
+![Image of Animal 1 on Eithon]
+(https://nms-seventh-fleet.github.io/images/image15.png)
+
+![Image of Animal 2 on Eithon]
+(https://nms-seventh-fleet.github.io/images/image1.png)
+
+![Image of Animal 3 on Eithon]
+(https://nms-seventh-fleet.github.io/images/image9.png)
+
+<p>On another planet, we found an enormous structure that served as a historical archive for the native Vy'keen.</p>
+
+![Image 1 of historical archive in the Vodanda-Ilib XII system ]
+(https://nms-seventh-fleet.github.io/images/image9.png)
```

Comment:
image9.png is the same as the "Animal 3" image. In fact, can we come up with a better naming scheme for image files? Maybe name them after the post date (e.g. 3420-10-03_001.png)

Reply:
resolved